examples/src/simple.rs

```diff
@@ -381,19 +381,12 @@ fn interpret_input(input: &str, sys: &mut System) -> bool {
         "system" => {
             writeln!(
                 &mut io::stdout(),
-                "System name:      {}",
-                sys.get_name().unwrap_or_else(|| "<unknown>".to_owned())
-            );
-            writeln!(
-                &mut io::stdout(),
-                "System version:   {}",
-                sys.get_version().unwrap_or_else(|| "<unknown>".to_owned())
-            );
-            writeln!(
-                &mut io::stdout(),
-                "System host name: {}",
+                "System name:           {}\nSystem kernel version: {}\nSystem OS version:     {}\nSystem host name:      {}",
```

Comment:
A bit hard to read. What do you think of:

```suggestion
                "System name:           {}\n\
                 System kernel version: {}\n\
                 System OS version:     {}\n\
                 System host name:      {}",
```?

Reply:
fixed in 81c7eb0